resume.go

```diff
@@ -0,0 +1,32 @@
+package dtls
+
+import (
+	"net"
+)
+
+// Export extracts dtls state and inner connection from an already handshaked dtls conn
+func (c *Conn) Export() (*State, net.Conn, error) {
+	return &c.state, c.nextConn, nil
```

Comment:
Should we copy the state here? Right now it gives users a chance to do weird things (by accident, but I could maybe see weird bugs happening)

Reply:
I've added a clone method to resolve this